MariaDB/server
# MDEV-33837: Curated lintian fixes for Debian

- [x] *The Jira issue number for this PR is: MDEV-33837*

## Description
These are curated list of Debian Lintian fixes to reduce them to minimal. They are all checked and noted as false positive. There still some left but they are not so trivial so they are not included. These are in new syntax that have been introduced in Debian 12.

Currently Salsa-CI can't build anything without **No space left in device** so no run there.

## Release Notes
Update Lintian notes

## How can this PR be tested?
Build in Sid or Debian 12 `debian/autobake-debs.sh` and run lintian with: `lintian  --display-info --pedantic --allow-root ../*.changes` List should like 10-12 items long.

## Basing the PR against the correct MariaDB version

## PR quality check
